dvl-a50/blueoshelper.py

```diff
@@ -10,7 +11,7 @@ def request(url):
     try:
         return urllib.request.urlopen(url, timeout=1).read().decode()
     except Exception as error:
-        print("Error in request: {0}: {1}".format(url, error))
+        logger.warning("Error in request: {0}: {1}".format(url, error))
```

Comment:
Any reason not to use f-strings for this?
```suggestion
        logger.warning(f"Error in request: {url}: {error}")
```

Reply:
just an oversight. fixing it.